### PR TITLE
feat: enable feature flags

### PR DESCRIPTION
**Pull request by UncleSamtoshi**
- create a feature flags context
- use firebase remote config to set the feature

### CODE REVIEW

File: app/graphql/feature-flags-context.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import React, { useState, createContext, useContext, useEffect } from "react"
+import remoteConfigInstance from "@react-native-firebase/remote-config"
+import { useAppConfig } from "@app/hooks"
+
+type FeatureFlags = {
+  deviceAccountEnabled: boolean
+}
+
+type RemoteConfig = {
```
Comment (Extheoisah) [inline app/graphql/feature-flags-context.tsx:9]: we could make this consistent by using only one type.
```suggestion
type RemoteConfig = FeatureFlags;
```
or just extend the `FeatureFlags` incase we want to add more properties to the RemoteConfig

**Review by Extheoisah — APPROVED** [review]
PR looks good to me. Just one comment

File: app/graphql/feature-flags-context.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  const {
+    appConfig: { galoyInstance },
+  } = useAppConfig()
+
+  useEffect(() => {
+    const fetchAndActivate = async () => {
+      try {
+        await remoteConfigInstance().fetchAndActivate()
+        const deviceAccountEnabled = remoteConfigInstance()
+          .getValue("deviceAccountEnabled")
```
Comment (nicolasburtey) [inline app/graphql/feature-flags-context.tsx:43]: value should be in a const

**Comment by nicolasburtey** [comment]
I believe this code has been merged already. so closing the PR but @UncleSamtoshi please reopen if I'm mistaken

**Pull request closed without merging**